fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/svarinn/MimeTypeExtensionMapper.java

```diff
@@ -1,5 +1,6 @@
 package no.difi.meldingsutveksling.ks.svarinn;
 
+import com.google.common.collect.HashBiMap;
 import com.google.common.collect.Maps;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.collect.Maps'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)